src/Sparrow/Extensions/EncodingsExtensions.cs

```diff
@@ -9,6 +9,9 @@ internal static class EncodingsExtensions
     {
         public static unsafe int GetBytes(this UTF8Encoding encoding, ReadOnlySpan<char> chars, Span<byte> bytes)
         {
+            if (chars.IsEmpty)
```

Comment:
Do we need to do the same in `GetString` ?

Reply:
Moved PR to 5.1